terraform/lambda/credential-issuer.tf

```diff
@@ -10,4 +10,8 @@ module "credential-issuer" {
   handler                = "uk.gov.di.ipv.lambda.CredentialIssuerHandler::handleRequest"
   function_name          = "${var.environment}-credential-issuer"
   role_name              = "${var.environment}-credential-issuer-role"
+
+  allow_access_to_user_issued_credentials_table = true
+  user_issued_credentials_table_policy_arn      = aws_iam_policy.policy-user-issued-credentials-table.arn
+  user_issued_credentials_table_name            = aws_dynamodb_table.user-issued-credentials.name
```

Comment:
I'd like to have the credential issuer handler have write only access to this table, see `policy-user-issued-credentials-table`, but we can do this in another story

Reply:
left this one - for now